src/components/Text/Text.tsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-interface TextProps {
+import { themeHelper, sxPropHelper, StyleProps } from '../../utils/styled';
+
+interface TextProps extends StyleProps {
     children: React.ReactNode;
     level: 1 | 2 | 3 | 4 | 5;
-    as?: React.ElementType;
 }
 
-const Text = styled.span<TextProps>((props) => ({
-    fontSize: (props.theme as any).fontSizes[props.level],
-    lineHeight: '125%',
-    fontFamily: 'Source Sans Pro, sans-serif',
-    fontWeight: 'normal',
-    color: (props.theme as any).colors.neutral[9],
-    '-webkit-font-smoothing': 'antialiased',
-    '-moz-osx-font-smoothing': 'grayscale',
-}));
+const Text = styled.h2<TextProps>(
```

Comment:
Is the text meant to be an `<h2>` tag? Copy paste error?

Reply:
Oops yes, good catch. Will fix

Reply:
🤔 span or div?

Reply:
`<div>` 👍

Reply:
Is that bad for accessibility or anything? I guess we could always change it using the `as` prop

Reply:
TBH I dunno which is more a11y friendly. We could run it through Axe and/or Wave to see if it throws an error.